stan/math/prim/scal/fun/log_falling_factorial.hpp

```diff
@@ -54,7 +54,7 @@ namespace math {
 template <typename T1, typename T2>
 inline typename return_type<T1, T2>::type log_falling_factorial(const T1 x,
                                                                 const T2 n) {
-  if (is_nan(x) || is_nan(n))
+  if (is_any_nan(x, n))
```

Comment:
[comment]
The changes look awesome! It does look like there's an impact on how we will write functions here on out.